### PR TITLE
LGTM Error Fix

### PR DESCRIPTION
**Pull request by GameDungeon**
## Overview: What does this pull request change?

Fixes many LGTM issues.

## Reviewer Checklist
- [ ] The PR title is descriptive enough for the changelog, and the PR is labeled correctly
- [ ] If applicable: newly added non-private functions and classes have a docstring including a short summary and a PARAMETERS section
- [ ] If applicable: newly added functions and classes are tested

### CODE REVIEW

File: manim/plugins/import_plugins.py [hunk 1]
```diff
@@ -36,7 +36,7 @@
         for lst in lists:
             exec(f"{lst.__name__}=lst")
             __all__.append(lst.__name__)
-    plugins_requested.remove(plugin.name)
-else:
-    if plugins_requested != []:
-        logger.warning("Missing Plugins: %s", plugins_requested)
+        plugins_requested.remove(plugin.name)
```
Comment (GameDungeon) [inline manim/plugins/import_plugins.py:39]:
```suggestion
    plugins_requested.remove(plugin.name)
```

File: manim/_config/utils.py [hunk 2]
```diff
@@ -1084,8 +1084,7 @@ def quality(self):
         for qual in constants.QUALITIES:
             if all([q[k] == constants.QUALITIES[qual][k] for k in keys]):
                 return qual
-        else:
-            return None
+        return None
```
Comment (Darylgolden) [inline manim/_config/utils.py:1087]: Wouldn't the correct way to "fix this" be to add a `break` statement above? Though for some reason this did not break Manim despite being here for a long time. I would prefer this "fix" to be taken out of this PR, and be fixed (if it is actually broken) in another PR.

File: manim/camera/camera.py [hunk 3]
```diff
@@ -187,8 +187,7 @@ def type_or_raise(self, mobject):
         for _type in self.display_funcs:
             if isinstance(mobject, _type):
                 return _type
-        else:
-            raise TypeError(f"Displaying an object of class {_type} is not supported")
+        raise TypeError(f"Displaying an object of class {_type} is not supported")
```
Comment (Darylgolden) [inline manim/camera/camera.py:190]: Same as above.

File: manim/cli/cfg/group.py [hunk 4]
```diff
@@ -200,8 +200,8 @@ def write(level: str = None, openfile: bool = False) -> None:
                             end="",
                         )
                         temp = input()
-                    else:
-                        default[key] = temp
+
```
Comment (Darylgolden) [inline manim/cli/cfg/group.py:203]: Same as above.

File: manim/plugins/import_plugins.py [hunk 5]
```diff
@@ -37,6 +37,6 @@
             exec(f"{lst.__name__}=lst")
             __all__.append(lst.__name__)
     plugins_requested.remove(plugin.name)
-else:
-    if plugins_requested != []:
-        logger.warning("Missing Plugins: %s", plugins_requested)
```
Comment (Darylgolden) [inline manim/plugins/import_plugins.py:42]: Same as above.

**Review by Darylgolden — CHANGES_REQUESTED** [review]
Since this PR is intended for cosmetic changes, I'd prefer the for else loops to be changed in another PR (they may really be broken right now).

**Comment by GameDungeon** [comment]
Well I asked in discord, and it was noted that it was a mistake. I think someone added the else due to the return in the for loop.

File: manim/_config/utils.py (hunk 2 above)
Comment (GameDungeon) [reply manim/_config/utils.py:1096]: As I've asked in discord, this is not broken, just written with bad practice. I think the original coder added the else when they saw the return in the loop.

**Review by Darylgolden — APPROVED** [review]
LGTM

File: manim/_config/utils.py (hunk 2 above)
Comment (ryanmccauley211) [reply manim/_config/utils.py:1096]: I think it looks fine to me as the else shouldn't have been there, and I think returning None is good as it is more explicit

**Review by ryanmccauley211 — APPROVED** [review]
LGTM!

**Pull request merged**